WD-Firmware/examples/ex-freertos/main.c

```diff
@@ -67,22 +67,26 @@
     1 tab == 4 spaces!
 */
 #ifdef D_USE_RTOSAL
-#include "rtosal_types.h"
-#include "rtosal_semaphore_api.h"
-#include "rtosal_task_api.h"
-#include "rtosal_mutex_api.h"
-#include "rtosal_queue_api.h"
-#include "rtosal_time_api.h"
-#include "rtosal_util_api.h"
-#include "psp_api.h"
-#include "task.h"     /* for tskIDLE_PRIORITY */
+   #include "rtosal_types.h"

```

Comment:
remove #include "rtosal_types.h"

Reply:
Irrelevant as this file is being abandoned